src/music/dmusic.cpp

```diff
@@ -9,8 +9,6 @@
 
 /** @file dmusic.cpp Playing music via DirectMusic. */
 
-#ifdef WIN32_ENABLE_DIRECTMUSIC_SUPPORT
```

Comment:
This (and WITH_UNISCRIBE) are for the benefit of MSVC as it's easier to add/remove a define in the project settings to test the possible variants.

Reply:
Applies to WITH_XAUDIO2 as well.

Reply:
That is an interesting use case. I guess that explains a bit why only MSVC-related files have this, and no other :D
With CMake, this would simply mean you regenerate the project files (it does that for you) if you add/remove a file (which is the same as add/remove a define, in CMake terms).

So I guess I can move this change to the CMake branch, and apply it once that hits. Does that work for you? (in other words, did I understand the use case correctly?) :D

Reply:
Well, I am proficient enough to cope no matter which way...

More seriously, adding/removing a define or a file isn't that much of a difference, especially if in CMake view, both looks identical to the users. So yes, I think keep it for now until CMake for real.